src/utils/components/MoveVMToFolderModal/MoveBulkVMsToFolderModal.tsx

```diff
@@ -7,17 +7,20 @@ import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTransla
 import { getNamespace } from '@kubevirt-utils/resources/shared';
 import { Popover, PopoverPosition, Stack, StackItem } from '@patternfly/react-core';
 
+import useRemoveFolderQuery from './hooks/useRemoveFolderQuery';
 import BulkVMsPopover from './BulkVMsPopover';
 import SelectedFolderIndicator from './SelectedFolderIndicator';
 
 type MoveBulkVMToFolderModalProps = {
+  allVMs: V1VirtualMachine[];
```

Comment:
I'm sure we should not let people move vms that have different namespaces. Do not make so much sense as folders are inside namespace.
Maybe we allow that now i don't remember but i don't think does make sense
This will let us remove all those `allVMs`
and fetch the namespace vms with `useVMsInNamespace`  inside `useRemoveFolderQuery` hook